autoload/vista/cursor.vim

```diff
@@ -203,6 +208,48 @@ function! vista#cursor#ShowDetailWithDelay() abort
         \ )
 endfunction
 
+" Show the detail of current tag/symbol under cursor.
+function! vista#cursor#FindDetail(_timer) abort
+  if empty(getline('.'))
+        \ || !exists('g:vista')
+        \ || !has_key(g:vista, 'functions')
+        \ || !has_key(g:vista, 'source')
+    call setbufvar(g:vista.source.bufnr, 'vista_nearest_scope', '')
+    call setbufvar(g:vista.source.bufnr, 'vista_nearest_symbol', '')
+    return
+  endif
+
+  let found = vista#util#BinarySearch(g:vista.raw, line('.'), 'line', '')
+  if empty(found)
+    let found = {}
+  endif
+
+  "echo found
+  let scope = vista#util#Trim(get(found, 'scope', ''))
+  if !empty(scope)
+    let scopeKind = get(found, 'scopeKind', '')
+    if !empty(scopeKind)
+      let scope = scopeKind.' '.scope
+    endif
+    let access = get(found, 'access', '')
+    if !empty(access)
+      let scope = scope.' '.access
+    endif
+  endif
+  let symbolname = vista#util#Trim(get(found, 'name', ''))
+  call setbufvar(g:vista.source.bufnr, 'vista_nearest_scope', scope)
+  call setbufvar(g:vista.source.bufnr, 'vista_nearest_symbol', symbolname)
```

Comment:
Hmm, I prefer using a Dict containing both the scope and symbol, which is also convient if we want to add more fields in it. For example, `call setbufvar(g:vista.source.bufnr, 'vista_cursor_info', {'scope': scope, 'symbol': symbol})`.